src/System.Private.ServiceModel/src/System/ServiceModel/Security/X509CertificateInitiatorClientCredential.CoreCLR.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Aside -- I have not heard a naming convention what we call "CoreCLR", but it looks like CoreFx is using this same naming pattern for similar purposes.  So I think this is the right file name for us.